addons/website_event_track/static/src/js/website_event_track_proposal_add_tags.js

```diff
@@ -0,0 +1,106 @@
+odoo.define('website_event_track.website_event_track_proposal_add_tags', function (require) {
+    'use strict';
+
+    var core = require('web.core');
+    var publicWidget = require('web.public.widget');
+
+    var _t = core._t;
+
+    publicWidget.registry.websiteEventProposalTags = publicWidget.Widget.extend({
+
+    selector: '.o_wetrack_proposal_add_tags_widget',
+
+    init: function (parent, options){
+        this.tagIds = [];
+        this._super(parent, options);
```

Comment:
You should stick to previous notation "this._super.apply(this, arguments)" since it works no matter the function arguments.

Or better yet the new one "this._super(...arguments)"